tracer/test/snapshots/MongoDbTests.SubmitsTraces_packageVersion=2_2.SchemaV0.verified.txt

```diff
@@ -89,6 +89,7 @@
       component: MongoDb,
       db.name: test-db,
       env: integration_tests,
+      language: dotnet,
```

Comment:
Why did `language: dotnet` start showing up? Is that expected?

Reply:
Yeah, basically these snapshots are out of date from when we started adding this to every span. These snapshots are only used when we do the "comprehensive testing" (i.e. hitting all package versions). So this is entirely incidental